angular/demo/bootstrap/src/app/samples/focustrack/focustrack.route.ts

```diff
@@ -42,19 +41,12 @@ import {ChangeDetectionStrategy, Component, effect, signal} from '@angular/core'
 export default class FocustrackComponent {
 	public readonly hasFocusApi = createHasFocus();
 	public readonly hasFocus = toAngularSignal(this.hasFocusApi.hasFocus$);
-	public readonly activeElements = signal<{tagName?: string; id?: string}[]>([]);
 	private readonly activeElement = toAngularSignal(activeElement$);
-
-	constructor() {
-		effect(
-			() => {
-				this.activeElements.update((elements) =>
-					elements.concat([{tagName: this.activeElement()?.tagName?.toLowerCase(), id: this.activeElement()?.id || undefined}]),
-				);
-			},
-			{allowSignalWrites: true},
-		);
-	}
+	public readonly activeElements = linkedSignal<Element | null, {tagName?: string; id?: string}[]>({
```

Comment:
as it is in the dev preview i would not use the linkedsignal (although it is cool)

Reply:
for demos we were already using preview things, as it's not part of an actual delivery